pages/content/reports/2024/index.md

```diff
@@ -0,0 +1,232 @@
+  - [Documentation for interoperable and widely available features](#documentation-for-interoperable-and-widely-available-features)
+  - [W3C WebDX web-features mapping of BCD keys into baseline features](#w3c-webdx-web-features-mapping-of-bcd-keys-into-baseline-features)
+  - [Replacing all "true" and "null" versions with real version numbers in BCD](#replacing-all-true-and-null-versions-with-real-version-numbers-in-bcd)
+  - [Web Security docs for MDN](#web-security-docs-for-mdn)
+  - [Updated Fetch documentation](#updated-fetch-documentation)
+  - [WebView compatibility data](#webview-compatibility-data)
+- Secured a second round of [investment from the Sovereign Tech Fund](https://openwebdocs.org/content/posts/sovereign-tech-fund/) to implement improved tooling and maintenance of the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) project and development of the [openwebdocs/mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) tool.
+
+Previous Open Web Docs Impact and Transparency Reports: [2023](https://openwebdocs.org/content/reports/2023/), [2022](https://openwebdocs.org/content/reports/2022/), [2021](https://openwebdocs.org/content/reports/2021/).
+
+<div class="callout">
+<p>Open Web Docs is a non-profit, strongly community-focused open source collective that uses its donations to employ a group of technical writers who are dedicated to writing and improving documentation for the web platform.</p>
+
+Donate today:
+
+- <a href="https://github.com/sponsors/openwebdocs">GitHub Sponsors</a>
+- <a href="https://opencollective.com/open-web-docs">Open Collective</a>
+
+Also, tell your company to support us with a membership!
+As a [member organization](https://openwebdocs.org/membership) you are entitled to further benefits in addition to supporting core web platform documentation and engineering. Reach out to [florian@openwebdocs.org](mailto:florian@openwebdocs.org) for more information.
+
+</div>
+
+## OWD maintainership by the numbers
+
+In 2024, Open Web Docs continued to work on improving and maintaining the following four essential projects for the web platform documentation ecosystem:
+
+- [mdn/content](https://github.com/mdn/content)
+- [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)
+- [openwebdocs/mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector)
+- [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features)
+
+Documentation about open web standards needs ongoing updating and maintenance as new web platform features are introduced and best practices change. We believe all of the above projects are essential information sources that web developers consult and trust on a daily basis. OWD believes that by contributing to these repositories, we have the biggest impact on open web platform documentation and we reach the majority of web developers worldwide.
```

Comment:
web-features is mentioned in several places in the report, but is a new thing in 2024 - maybe this would be the right place to introduce what it is, and why OWD determined it's a useful conduit to its mission.